README.MD

```diff
@@ -246,28 +114,30 @@ spec:
     storageOSOperatorNamespace: storageos-operator-old
 ```
 
-For an example config file, see:
+For an example config file, see `config/samples/_v1_kubectlstorageosconfig.yaml`.
 
-`config/samples/_v1_kubectlstorageosconfig.yaml`
+## Enable TLS
 
-## TLS Enablement
+### Install ETCD and StorageOS with TLS enabled
 
-### Install ETCD and StorageOS With TLS Enabled
-
-`kubectl storageos install --include-etcd --etcd-tls-enabled`
+```bash
+kubectl-storageos install --include-etcd --etcd-tls-enabled
```

Comment:
I think `kubectl storageos` is the right syntax

Reply:
Reverted